lake-manifest.json

```diff
@@ -1,37 +1,37 @@
-{"version": 5,
+{"version": 6,
  "packagesDir": "lake-packages",
```

Comment:
Is modifying the manifest here intentional?

Reply:
No. :-( I've reverted.